slides/readme.md

```diff
@@ -1,10 +1,18 @@
 # Slides
-Build these slides with [https://github.com/marp-team/marp-cli](Marp CLI)
-
+Build these slides with [Marp CLI](https://github.com/marp-team/marp-cli)
 
+Installing Marp
+```shell
+npm install --save-dev @marp-team/marp-cli
+```
+Running slide-deck
 
 ```shell
-marp --allow-local-files --pdf ./slides.md
+npx marp ./slides -s
+```
+Converting to PDF
```

Comment:
Added steps for running slide-deck server.
– I may _not_ have this right; if not, please feedback

Reply:
Maybe add that the server defaults to `8080` (I think)